tests/phpunit/tests/media.php

```diff
@@ -4951,16 +5021,18 @@ public function test_wp_get_loading_optimization_attributes_header_block_templat
 	 */
 	public function test_wp_get_loading_optimization_attributes_incorrect_loading_attrs() {
 		$attr                  = $this->get_width_height_for_high_priority();
+		$attr['decoding']      = 'async';
 		$attr['loading']       = 'lazy';
 		$attr['fetchpriority'] = 'high';
 
 		$this->assertEqualSetsWithIndex(
 			array(
+				'decoding'      => 'async',
 				'loading'       => 'lazy',
 				'fetchpriority' => 'high',
 			),
 			wp_get_loading_optimization_attributes( 'img', $attr, 'test' ),
-			'This should return both lazy-loading and high fetchpriority, but with doing_it_wrong message.'
+			'This should return lazy-loading, async decoding and high fetchpriority, but with doing_it_wrong message.'
```

Comment:
This can be reverted as the test is only focused on the problem with `loading="lazy"` and `fetchpriority="high"` present together.